is-with-analytics/README.md

```diff
@@ -43,19 +45,18 @@ kubectl config set-context $(kubectl config current-context) --namespace=wso2
 Create a Kubernetes Secret named `wso2creds` in the cluster to authenticate with the WSO2 Docker Registry, to pull the required images.
 
 ```
-kubectl create secret docker-registry wso2creds --docker-server=docker.wso2.com --docker-username=<username> --docker-password=<password> --docker-email=<email>
+kubectl create secret docker-registry wso2creds --docker-server=docker.wso2.com --docker-username=<FT_USERNAME> --docker-password=<FT_PASSWORD> --docker-email=<FT_USERNAME>
```

Comment:
This is not FT specific, any WSO2 account with a valid subscription is acceptable here.